platforms/android/src/jp/wizcorp/phonegap/plugin/WizAssets/HttpToFile.java

```diff
@@ -0,0 +1,133 @@
+package jp.wizcorp.phonegap.plugin.WizAssets;
+import java.io.*;
+import java.net.Authenticator;
+import java.net.HttpURLConnection;
+import java.net.PasswordAuthentication;
+import java.net.URL;
+import android.util.Log;
+
+public class HttpToFile {
```

Comment:
Why not letting this class' methods throw? And catch on `WizAssetsPlugin`'s side?